source/aws_profile.c

```diff
@@ -1039,15 +1039,15 @@ static bool s_parse_property_continuation(
      * A continuation without a current property is bad
      */
     if (context->current_profile == NULL || context->current_property == NULL) {
-        AWS_LOGF_WARN(AWS_LS_SDKUTILS_PROFILE, "Property continuation seen outside of a current property");
+        AWS_LOGF_DEBUG(AWS_LS_SDKUTILS_PROFILE, "Property continuation seen outside of a current property");
```

Comment:
This is a fatal error and it should not occur frequently unless your profile is messed up. Maybe: 
```suggestion
        AWS_LOGF_ERROR(AWS_LS_SDKUTILS_PROFILE, "Property continuation seen outside of a current property");
```